File: backend/tests/factories/environment.py
import factory
from faker import Faker

from app.configuration.environments.model import Environment

fake = Faker()


class EnvironmentFactory(factory.alchemy.SQLAlchemyModelFactory):
    class Meta:
        model = Environment

    id = factory.Faker("uuid4")
    name = "dev"
    acronym = factory.Sequence(lambda _: fake.unique.word())
    context = "environment_context{{}}"
    is_default = False

File: sdk-go/models/environment.go
package models

type Environment struct {
	ID        string `json:"id"`
	Name      string `json:"name"`
	Acronym   string `json:"acronym"`
	Context   string `json:"context"`
	IsDefault bool   `json:"is_default"`
}

type EnvironmentCreate struct {
	Name    string `json:"name"`
	Acronym string `json:"acronym"`
	Context string `json:"context"`
}
